### PR TITLE
Switch from yarn to pnpm for hardhat external tests

### PR DESCRIPTION
**Pull request by r0qs**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1326,16 +1326,16 @@ jobs:
           name: Install dependencies
           command: |
             cd hardhat
-            yarn
+            pnpm install
```
Comment (nikola-matic) [inline .circleci/config.yml:1329]: Huh, I thought this had to be installed on the circleci image used to run the test. So it just comes preloaded?

Comment (r0qs) [reply .circleci/config.yml:1329]: It is installed by default in the `cimg/node` docker image from CircleCI: https://github.com/CircleCI-Public/cimg-node/blob/main/18.16/Dockerfile#L30

Comment (nikola-matic) [reply .circleci/config.yml:1329]: Hahaha, well then, I guess this was significantly easier to do than I thought :)

**Review by nikola-matic — APPROVED** [review]

**Comment by nikola-matic** [comment]
Pretty sure you can merge, don't think there's need to wait on @cameel for this one.

**Comment by cameel** [comment]
Looks good in any case.

**Pull request merged**